### DISCUSSION TITLE
Day Picker component stays Open even after selecting a date

### DISCUSSION THREAD

**Discussion by zain-getdirectholidays** [gpbl/react-day-picker#2221] [Support]
## Expected Behavior

I expect DayPicker should stay close when we haven't interacted with it.

## Actual Behavior

But it stays open all the time. even after selecting a date, it doesn't get closed.

## Screenshots
This is how it looks when I add the DayPicker component
<img width="573" alt="Screenshot 2024-06-24 at 4 13 28 PM" src="https://github.com/gpbl/react-day-picker/assets/169983176/0dcddedd-2639-4d97-9288-844bd8487aab">

## Your Environment

- NextJS 14 with TypeScript App Router
- Pakistan Time Zon
- Browser: Google Chrome
- Version [e.g. 22]:
- Operating System [e.g. iOS, Windows]: MacOS

**Comment by Dchamizom9108** [comment]
I use a handler funtion for show/hide

  const handleDateChange = (date) => {
    setUserData({
      ...userData,
      effectiveDate: date,
    });
    setShowCalendar(false);
  };
